packages/flutter/lib/src/rendering/child_view.dart

```diff
@@ -10,13 +11,28 @@ import 'package:mojo_services/mojo/ui/layouts.mojom.dart' as mojom;
 import 'package:mojo_services/mojo/ui/view_provider.mojom.dart' as mojom;
 import 'package:mojo_services/mojo/ui/views.mojom.dart' as mojom;
 import 'package:mojo/application.dart';
+import 'package:mojo/core.dart' as core;
 import 'package:mojo/mojo/service_provider.mojom.dart' as mojom;
 
 import 'box.dart';
 import 'object.dart';
 
-// TODO(abarth): Populate the view host.
-final mojom.ViewHost _viewHost = null;
+mojom.ViewHostProxy _initViewHostProxy() {
+  try {
+    return new mojom.ViewHostProxy.fromHandle(
+      new core.MojoHandle(internals.takeViewHostHandle()));
+  } catch (e) {
+    return null;
```

Comment:
do we really want to silently catch all exceptions here? Or just one specific one?